tutorials/hyperspectral/Work-With-Hyperspectral-Data-In-R.Rmd

```diff
@@ -105,33 +112,32 @@ library(rhdf5)
 library(rgdal)
 
 # set working directory to ensure R can find the file we wish to import and where
-# we want to save our files. Be sure to move the downloaded into your working directory!
-wd="~/Desktop/Hyperspectral_Tutorial/"
+# we want to save our files. Be sure to move the download into your working directory!
+wd="~/Desktop/Hyperspectral_Tutorial/" #This will depend on your local environment
 setwd(wd)
 
+# Define the file name to be opened
+f <- paste0(wd,"NEON_hyperspectral_tutorial_example_subset.h5")
 ```
 
 <div id="ds-dataTip" markdown="1">
 <i class="fa fa-star"></i> **Data Tip:** To update all packages installed in 
 R, use `update.packages()`.
 </div>
 
-```{r view-file-strux }
-
-# Define the file name to be opened
-f <- paste0(wd,"NEONDSImagingSpectrometerData.h5")
+```{r view-file-strux, eval=FALSE, comment=NA}
 # look at the HDF5 file structure 
-h5ls(f,all=T) 
-
+View(h5ls(f,all=T))
 ```
 
-When you look at the structure of the data, take note of the `map info` dataset, 
-the `spatialInfo` group, and the `wavelength` and `Reflectance` datasets. The 
-`spatialInfo` folder contains the spatial attributes of the data including its 
-Coordinate Reference System (CRS). The CRS documents how the data are physically 
-located on the Earth. The `wavelength` dataset contains the middle wavelength 
-values for each band in the data. The reflectance dataset contains the image data 
-that we will use for both data processing and visualization. 
+When you look at the structure of the data, take note of the `map info` dataset,
+the `Coordinate_System` group, and the `wavelength` and `Reflectance` datasets. The
+`Coordinate_System` folder contains the spatial attributes of the data including its
+EPSG Code, which is easily converted to a Coordinate Reference System (CRS). 
+The CRS documents how the data are physically located on the Earth. The `wavelength` 
+dataset contains the middle wavelength values for each band in the data. The 
+`Reflectance` dataset contains the image data that we will use for both data processing 
+and visualization.
 
 More Information on raster metadata:
 
```

Comment:
I think we should update lines 138 &139 to list the actual title of the Tutorials and then have description of why it is references outside the hyperlink: 

* <a href="{{ site.baseurl }}/raster-data-r" target="_blank"> Raster Data in R - The Basics</a>: Metadata to understand when working with raster data

Reply:
Good point. I have made these changed in this latest commit.